buf.lock

```diff
@@ -0,0 +1,9 @@
+# Generated by buf. DO NOT EDIT.
```

Comment:
do we need to check-in this `lock` file?

Reply:
Yes, it is supposed to be chedked in